Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dto/ExamDto.java

```diff
@@ -28,6 +28,8 @@
     private String status;
     @Column("module_id")
     private Integer moduleId;
+    @Column("total_points")
+    private Integer totalPoints;
```

Comment:
ich benutze private total_points; soll ich dies ändern oder du?

Reply:
Du, unterstriche sollten eigentlich nicht in die variablen namen